js/adapt-contrib-resourcesView.js

```diff
@@ -49,7 +49,20 @@ define(function(require) {
         },
 
         onResourceClicked: function(event) {
-            window.top.open($(event.currentTarget).data("href"));
+            var data = $(event.currentTarget).data();
+
+            if ('document' !== data.type) {
+              window.top.open($(event.currentTarget).data("href"), 'MyFile.jpg');
```

Comment:
The `data.href` contains the URL already?  I'm not sure the second parameter is required, as we could have PDFs, text files, etc.